### PR TITLE
CVE-2004-1834

### PR DESCRIPTION
**Pull request by JosefLenz**
history project round 1

### CODE REVIEW

**Comment by JMdoubleU** [comment]
It might be helpful to give a general description of what `mod_disk_cache` is in the description (though maybe it's self explanatory...). Was this only an issue in some configurations with caching enabled?

Can you double-check the fix commit for this? The currently listed commit was on 2002-11-17, but it seems the [issue was reported on 2004-03-03](https://marc.info/?l=bugtraq&m=107981737322495&w=2).

**Comment by JosefLenz** [comment]
Good point, ill see if i can find break it down a bit better, I believe Mod_disk_cache was the setting in the configuration that when enabled caused the cache to save to the disk.good catch on the date, ill correct that.

File: cves/CVE-2004-1834.yml [hunk 1]
```diff
@@ -20,19 +20,23 @@ announced_instructions: |
...
+    read to anyone with some programming experience. We can always pull up the NVD
+    description later to get more technical.
+
+    Try to still be specific in your description, but remove Chromium-specific
+    stuff. Remove references to versions, specific filenames, and other jargon
+    that outsiders to Chromium would not understand. Technology like "regular
+    expressions" is fine, and security phrases like "invalid write" are fine to
+    keep too.
+description: |
+    The mod_disk_cache is a tag that sets the program to use disk based storing of cashes,
```
Comment (cxd3796) [inline cves/CVE-2004-1834.yml:36]: caches*

File: cves/CVE-2004-1834.yml [hunk 2]
```diff
@@ -20,19 +20,23 @@ announced_instructions: |
...
+    this casheing in included client headers on the users local hard disk. Users can then
```
Comment (cxd3796) [inline cves/CVE-2004-1834.yml:37]: caching

File: cves/CVE-2004-1834.yml [hunk 3]
```diff
@@ -158,8 +207,10 @@ lessons:
     applies:
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: the caching function should not have been given access to secure information
+          nor should any part of the system that stores credential have been given
```
Comment (cxd3796) [inline cves/CVE-2004-1834.yml:212]: credentials*

File: cves/CVE-2004-1834.yml [hunk 4]
```diff
@@ -200,4 +251,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The initial developers did not understand either the funcionality of the tools
+    they were using for cacheing or the full content of the header of which they
```
Comment (cxd3796) [inline cves/CVE-2004-1834.yml:256]: caching*

File: cves/CVE-2004-1834.yml [hunk 5]
```diff
@@ -200,4 +251,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```
Comment (cxd3796) [inline cves/CVE-2004-1834.yml:254]: Could the developers have approached the design of the tools in a different manner that would have mitigated against this vulnerability?

**Pull request merged**